includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -633,8 +630,8 @@ private function process_webhook_dispute_updated( $event_body ) {
 			__( '%1$s. See <a href="%2$s">dispute overview</a> for more details.', 'woocommerce-payments' ),
 			$message,
 			add_query_arg(
-				[ 'id' => $dispute_id ],
-				admin_url( 'admin.php?page=wc-admin&path=/payments/disputes/details' )
+				[ 'id' => $charge_id ],
+				admin_url( 'admin.php?page=wc-admin&path=/payments/transactions/details' )
```

Comment:
Either

```suggestion
				admin_url( 'admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails' )
```

Or consistent with `compose_dispute_url()`:
```
add_query_arg(
	[
		'page' => 'wc-admin',
		'path' => urlencode( '/payments/transactions/details' ),
		'id' => $charge_id
	],
	admin_url( 'admin.php' )
)
```

Reply:
I've opened a followup issue for this: #7399.